src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -0,0 +1,229 @@
+
+        # Number of Clients: 13
+        wireless_client_count_capture = re.compile(r"^Number\s+of\s+Clients:\s+(?P<wireless_client_count>\d+)$")
+        # MAC Address    AP Name                                        Type ID   State             Protocol Method     Role
+        wireless_client_info_header_capture = re.compile(
+            r"^MAC\s+Address\s+AP\s+Name\s+Type\s+ID\s+State\s+Protocol\s+Method\s+Role$")
+        # -------------------------------------------------------------------------------------------------------------------------
+        delimiter_capture = re.compile(
+            r"^-------------------------------------------------------------------------------------------------------------------------$")
+        # 58bf.ea58.b399 b80-72-cap16                                 WLAN 17   Run               11ac     Dot1x      Local
+        wireless_client_info_capture = re.compile(
+            r"^(?P<mac_address>\S+)\s+(?P<ap_name>\S+)\s+(?P<type>\S+)\s+(?P<id>\d+)\s+(?P<state>.*)\s+(?P<protocol>\S+)\s+(?P<method>(Dot1x|MAB))\s+(?P<role>\S+)$")
+        # Number of Excluded Clients: 2
+        wireless_excluded_client_count_capture = re.compile(
+            r"^Number\s+of\s+Excluded\s+Clients:\s+(?P<wireless_excluded_client_count>\d+)$")
+        # MAC Address    AP Name                          Type ID   State              Protocol Method
+        exclude_client_header_capture = re.compile(
+            r"^MAC\s+Address\s+AP\s+Name\s+Type\s+ID\s+State\s+Protocol\s+Method$")
+        # ------------------------------------------------------------------------------------------------
+        delimiter_2_capture = re.compile(
+            r"^------------------------------------------------------------------------------------------------$")
+        # 58bf.ea8a.20e6 b80-61-cap20                   WLAN 17   Excluded           11ac     Dot1x
+        wireless_excluded_clients_info_capture = re.compile(
+            r"^(?P<mac_address>\S+)\s+(?P<ap_name>\S+)\s+(?P<type>\S+)\s+(?P<id>\d+)\s+(?P<state>\S+)\s+(?P<protocol>\S+)\s+(?P<method>\S+)$")
+
+        include_index = 0
+        exclude_index = 0
+
+        for line in out.splitlines():
+            line = line.strip()
+            # Number of Clients: 13
+            if wireless_client_count_capture.match(line):
+                wireless_client_count_capture_match = wireless_client_count_capture.match(line)
+                groups = wireless_client_count_capture_match.groupdict()
+                if not show_wireless_client_summary_dict.get('wireless_client_summary', {}):
+                    show_wireless_client_summary_dict['wireless_client_summary'] = {}
+                wireless_client_count = int(groups['wireless_client_count'])
+                show_wireless_client_summary_dict['wireless_client_summary'][
+                    'wireless_client_count'] = wireless_client_count
+                continue
```

Comment:
if you are using elif no need for continue? As it will continue loop anyway?

Reply:
This is to support this design pattern https://github.com/CiscoTestAutomation/genieparser/pull/256#discussion_r493013132 e.g. a future raise after this.

Reply:
confirmed